test/specs/controller.bar.tests.js

```diff
@@ -1519,4 +1519,152 @@ describe('Bar controller tests', function() {
 			};
 		});
 	});
+
+	describe('Bar thickness with a category scale', function() {
```

Comment:
Is there any way to generalize these 2 tests and avoid dependencies between the bar controller and category/time scales?